image-layout.md

```diff
@@ -2,7 +2,11 @@
 
 The OCI Image Layout is a slash separated layout of OCI content-addressable blobs and [location-addressable](https://en.wikipedia.org/wiki/Content-addressable_storage#Content-addressed_vs._location-addressed) references (refs).
 This layout MAY be used in a variety of different transport mechanisms: archive formats (e.g. tar, zip), shared filesystem environments (e.g. nfs), or networked file fetching (e.g. http, ftp, rsync).
-Given an image layout a tool can convert a given ref into a runnable OCI Image Format by finding an appropriate manifest from the manifest list, unpacking the [filesystem layers](layer.md) in the correct order, and then converting the image configuration into an OCI Runtime config.json.
+
+Tooling can:
+* convert a given ref into a runnable OCI Image Format by finding an appropriate manifest from the manifest list
+* apply the [filesystem layers](layer.md) per the [manifest layers ordering](manifest.md#image-manifest-property-descriptions)
+* convert the image configuration into an [OCI Runtime config.json](https://github.com/opencontainers/runtime-spec)
```

Comment:
All of these steps are required to create a runable bundle.  How about:

> Given an image layout and a ref, a tool can create a runnable [OCI Runtime Specification bundle][bundle] by:
>
> 1. Following the ref to find a [manifest](link), possibly via a [manifest list](link),
> 2. Unpacking the [filesystem layers](layer.md) in the correct order, and
> 3. Converting the [image configuration](link) into an [OCI Runtime Specification `config.json`][runtime-config].

[bundle]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/bundle.md
[runtime-config]: https://github.com/opencontainers/runtime-spec/blob/v1.0.0-rc2/config.md